scripts/download.sh

```diff
@@ -11,11 +11,12 @@ then
   fi
 elif [ "$CODECOV_USE_PYPI" == "true" ];
 then
-  if ! pip install codecov-cli"$([ "$CODECOV_VERSION" == "latest" ] && echo "" || echo "==$CODECOV_VERSION" )"; then
+  if ! pip install "${CODECOV_CLI_TYPE}-cli$([ "$CODECOV_VERSION" == "latest" ] && echo "" || echo "==$CODECOV_VERSION")"; then
     exit_if_error "Could not install via pypi."
     exit
   fi
-  CODECOV_COMMAND="codecovcli"
+
+CODECOV_COMMAND="${CODECOV_CLI_TYPE}cli"
```

Comment:
can we consider having these things be the same name (e.g. `pip install codecov-cli` and `codecov-cli` be the command?

probably out of scope, but wanted to bring this up

Reply:
Also note that the executable name we settled on for prevent is `sentry-prevent-cli`